recipe/meta.yaml

```diff
@@ -1,24 +1,32 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
-
-# Jinja variables help maintain the recipe as you'll update the version only here.
-# Using the name variable with the URL in line 13 is convenient
-# when copying and pasting from another recipe, but not really needed.
 {% set name = "vega-cli" %}
-{% set version = "5.6.0" %}
+{% set version = "5.17.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 build:
   number: 0
-  skip: true  # [win]
 
 requirements:
   build:
-    - yarn
+    - "{{ compiler('c') }}"
+    - "{{ compiler('cxx') }}"
+    - python *
+    - pkg-config
+  host:
+    - nodejs 14.*
+    - cairo     # [not win]
+    - freetype  # [not win]
+    - giflib    # [not win]
+    - glib      # [not win]
+    - jpeg      # [not win]
+    - libpng    # [not win]
+    - pango     # [not win]
+    - yarn      # [not win]
   run:
-    - nodejs
+    - nodejs 14.*
```

Comment:
Why does this require nodejs 14.*?

Reply:
This package includes node-canvas which is binary and thus pinned to the major release of `nodejs`. On Linux and MacOS we could build easily for more node versions but on Windows I wasn't able to get the build going and thus I'm using the pre-built node-canvas there (which is only available for node 14.*).

Reply:
OK. Thanks